lib/index.js

```diff
@@ -183,6 +183,48 @@ function _ensure_cheerio(value) {
     return value;
 }
 
+class DictionaryAsset extends BaseAsset {
+    constructor() {
+        super();
+        this._object_type = "DictionaryObjectModel";
+        this._content_type = "text/html";
+    }
+
+    set_word (value) { this._word = value; }
+    set_definition (value) { this._definition = value; }
+    set_part_of_speech (value) { this._part_of_speech = value; }
+    set_tags (value) { this._tags = value; }
+
+    to_metadata() {
+        const metadata = super.to_metadata();
+        Object.assign(metadata, {
+            'word': this._word,
+            'definition': this._definition,
+            'part-of-speech': this._part_of_speech,
```

Comment:
in the other classes, we use camelCase for metadata keys. s/part-of-speech/partOfSpeech